enclave_manager/avalon_enclave_manager/graphene/zmq_comm.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python3
+
+import sys
+import os
+import subprocess
+import zmq
+import logging
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler(sys.stdout))
+
+# -------------------------------------------------------------------------
+
+
+class ZmqCommunication():
```

Comment:
Can we move this module to common/python, Since it is utility function. It can be used by other modules.

Reply:
Yes. We can move this file to common/python/utility